src/Microsoft.DotNet.BuildTools.TestSuite/runtime.json

```diff
@@ -69,6 +69,7 @@
             "netcoreapp1.0": [
                 "win7-x86",
                 "win7-x64",
+                "win10-arm64",
```

Comment:
Just wanted to confirm that this is the root of the arm64 leg of windows from https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json#L68-L69 .